Encryption/solution/main.py

```diff
@@ -0,0 +1,69 @@
+import base64
+import re
+import datetime
+
+import FileManager
+import os
+
+import numpy
+import numpy as np
+from PIL import Image
+#ex1
+# FileManager.DecodeFilesNameBase32("D:\pandaCourse\Python\covidEX\question\ex1")
```

Comment:
You didn't need to lib, its simple enough (and its an unknown and not popular lib, best to avoid these. Last update was in 2020 and it has 1 star )